docker-compose.yml

```diff
@@ -86,6 +86,26 @@ services:
     depends_on:
       - mongodb
 
+  devctl:
+    container_name: devctl
+    hostname: devctl
+    image: hemsaw/devctl:experimental
```

Comment:
It should be the latest tag; the experimental tag is run for internal testing. I will be pushing the first init and official pull after the device file is tested.